lib/tasks_private/autosde_oas.rake

```diff
@@ -7,7 +7,7 @@ namespace :autosde do
 
       root = ManageIQ::Providers::Autosde::Engine.root # ManageIQ::Environment::APP_ROOT
       # parent folder for generated stuff ( contains input spec file)
-      default_folder = "app/models/manageiq/providers/autosde/storage_manager/openapi_client"
+      default_folder = "lib/autosde_oas_client"
```

Comment:
I forgot to update this when I moved the client out of `app/models` that was my bad